.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+dist: trusty
+sudo: true
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
```

Comment:
@cmnbroad So I forgot about this, but if you want to upload archives you'll have to include the password for the artifactory as an encrypted variable in the yml